source/components/other-utils.md

```diff
@@ -50,6 +50,23 @@ window.addEventListener(
 )
 ```
 
+Vue SFV example:
+```
+<template>
+<q-input @keyup="save(submission)" v-model="submission"  />
+</template
+<script>
+export default {
+ import { debounce } from 'quasar'
```

Comment:
import statement inside the export generates error. it should be outside of the exported object.